src/custom-element.js

```diff
@@ -40,6 +40,7 @@ import {
 import * as dom from './dom';
 import {setStyle, setStyles} from './style';
 import {LayoutDelayMeter} from './layout-delay-meter';
+import {ResourceState} from './service/resource';
```

Comment:
I added it to prevent `toggleFallback` on after element has been unlaid out. It would be great if there's an internal state for it. But I can't find one.